lib/github/ldap.rb

```diff
@@ -281,6 +292,29 @@ def configure_membership_validation_strategy(strategy = nil)
         end
     end
 
+    # Internal:  Set the user search strategy that will be used by
+    #            Domain#user?.
+    #
+    # strategy - Can be either 'default' or 'global_catalog'.
+    #            'default' strategy will search the configured
+    #            domain controller with a search base relative
+    #            to the controller's domain context.
+    #            'global_catalog' will search the entire forest
+    #            using Active Directory's Global Catalog
+    #            functionality.
+    def configure_user_search_strategy(strategy)
+      @user_search_strategy = begin
+        case strategy.to_s
```

Comment:
Don't need the `begin ... end` block here, `case ... end` is essentially a block.